Geometry/CMSCommonData/python/cmsExtendedGeometry2023D3XML_cfi.py

```diff
@@ -91,7 +91,7 @@
         'Geometry/HcalCommonData/data/hcalouteralgo.xml',
         'Geometry/HcalCommonData/data/hcalforwardalgo.xml',
         'Geometry/HcalCommonData/data/PhaseII/hcalSimNumbering.xml',
-        'Geometry/HcalCommonData/data/PhaseII/HGCal/hcalRecNumberingRebuild.xml',
+        'Geometry/HcalCommonData/data/PhaseII/hcalRecNumberingRebuild.xml',
```

Comment:
... for the backing hadron calorimeter alone. Since this PR is comparing to Silicon-only simclusters it shouldn't be noticeable.

What *is* noticeable is that the HE hits have been completely removed from the Hcal digitization and reconstruction workflow, and should no longer show up in PF.